packages/core/src/embedded-checkout/resizable-iframe-creator.spec.ts

```diff
@@ -107,7 +107,7 @@ describe('ResizableIframeCreator', () => {
 
         const frame = await iframeCreator.createFrame(url, 'checkout');
 
-        expect(frame.allowPaymentRequest).toEqual(true);
+        expect((frame as any).allowPaymentRequest).toEqual(true);
```

Comment:
Any?

Reply:
Yes had to do this since the `allowPaymentRequest`  property is supported only in chrome, and typescript doesn't ship with it anymore. So had to do this, will have a separate item to review this use case and if we can refactor this 😄